Lombiq.VueJs.Samples/Lombiq.VueJs.Samples.csproj

```diff
@@ -1,15 +1,13 @@
 <Project Sdk="Microsoft.NET.Sdk.Razor">
 
+  <Import Condition="'$(NuGetBuild)' != 'true'" Project="..\..\..\Utilities\Lombiq.NodeJs.Extensions\Lombiq.NodeJs.Extensions\Lombiq.NodeJs.Extensions.props" />
+  <Import Project="..\..\Lombiq.VueJs\Lombiq.VueJs\Lombiq.VueJs.props" />
```

Comment:
This is me being forwards-thinking (perhaps too much for my own good). Adding 4 lines instead of 2 is not the end of the world, especially because you'd just copy-paste it [from the readme](https://github.com/Lombiq/Orchard-Vue.js/blob/b897ddb542249824df71f680c8d34222e78463fd/Readme.md#prerequisites) anyway. But if we ever come up with a different library project that also builds on NE, then following your suggestion we wouldn't be able to include both in the same project as that would lead to importing NE twice. In other words moving NE targets inside the LV targets would make it non-composable.

Yes I know, highly theoretical, but I feel if this scenario ever came through then taking the import out of the LV props would be extremely painful at that point..

Reply:
Ever heard of YAGNI? 😁 

Jokes aside, I can understand your point. I do have a different opinion on the matter though, because for this to become a problem, two things have to happen at once:
1. Another library project based on NE.
2. A consumer project that uses **both** library projects at the same time.

The combined probability of these two occurring at the same time is even lower than just that of 1.

In my opinion, the value of not having to import two files exceeds the fact that it's 1 instead of 2 lines to copy each - it's the simpler mental model that this allows for. Consumer `Lombiq.VueJs`, **I don't have to know or care** about this project using Node.js Extensions.

I've already made the same decision, when I hid away `NPM Targets` in the internals of how NE works. The user does not need to know about NPM Targets at all, and I think that in itself is a value.

To conclude:
> taking the import out of the LV props would be extremely painful at that point

I don't think so 😀

Reply:
I don't think YAGNI applies here, because we have already needed "it" once, "it" being a library that builds on top of NE.

> Consumer Lombiq.VueJs, I don't have to know or care about this project using Node.js Extensions.

I don't think that's actually true. You still have to know and care about NE, to fulfill its prerequisites during developer setup.

> > taking the import out of the LV props would be extremely painful at that point
>
> I don't think so 😀

If we have to go back on  this change once a lot of projects use LV then all of them would have to be edited, and backwards compatibility is broken. Sounds like a nightmare scenario to me.